packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1037,6 +1038,23 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			}
 		}
 
+		async function onCopy(event: any) {
+			const copiedContent = editor.selection.getContent();
+
+			// We need to remove timestamps from the image URLs while copying
+			// because some offline edtors do not show the image if there is
+			// an extra ?t= in it's path.
+			// Related to - https://github.com/laurent22/joplin/issues/4602
+			const removeTimestampFromUrl = (url: string) => {
+				const index = url.lastIndexOf('?t=');
```

Comment:
This is a fragile check. What if later on more parameters are added and the string is now "&t="? Your check will now fail and since you didn't provide any unit tests, we won't know that it's broken. You could in fact easily create a unit test for that particular function.

Reply:
Improved the logic of the check.

> You could in fact easily create a unit test for that particular function.

I actually couldn't find the file where tests for `TinyMCE` were written. If you're referring to only testing the `removeTimestampFromUrl`. Should I create a different file in the utils for this function and then test it?